testsuite/docs/cucumber-steps.md

```diff
@@ -613,6 +625,44 @@ For example:
   When I call actionchain.add_package_install()
 ```
 
+<a name="b13" />
+
+#### Virtualization
+
+* Create a test virtual machine on a given host
+
+The virtual machine is created without SUSE Manager, directly on the virtual host
+using `qemu-img` and `virt-install`
+
+```cucumber
+  When I create "test-vm" virtual machine on "virt-server"
+```
+
+* Assert a given virtual machine to be in a given state
+
+The possible states are `shut off`, `running` and `paused`.
+
+```cucumber
+  Then I should see "test-vm" virtual machine "shut off" on "virt-server"
+```
+
+* Waiting for a given virtual machine OS to be started
+
+```cucumber
+  When I wait until virtual machine "test-vm" on "virt-server" OS is started
+```
```

Comment:
Remove "OS " ?

Suggested: When I wait until virtual machine "test-vm" is started on "virt-server"

Reply:
that one needs not to conflict with the other step that checks the state of a VM regardless of the state of the OS in it.